src/Copy/Text.elm

```diff
@@ -464,7 +464,7 @@ t key =
             "I often wonder how I got here. I got married to this charming man in the legal profession and we had children. But it was after I got married that I was systematically and unwittingly socially isolated, psychologically, emotionally, financially abused and controlled."
 
         SplitterAffirmation ->
-            "If you are experiencing economic abuse, you are not alone. There are\npeople and\u{00A0}organisations who understand and who can help."
```

Comment:
I can't recall what the non-breaking space is doing here now, they aren't usually in the middle of a line.

Reply:
Probably just making the text symmetrical?